contribs/gmf/less/vars.less

```diff
@@ -30,3 +30,4 @@
 
 @brand-primary: #b1c2b5;
 @brand-secondary: #d3e5d7;
+@font-size-small: ceil(@font-size-base * 0.9);
```

Comment:
I think that we shouldn't use ceil and floor to be ready for bootstrap 4...
https://github.com/twbs/bootstrap/pull/17021